### ISSUE TITLE
PF4: React Virtualized table follow up issue

### ISSUE THREAD

**Issue by priley86** [patternfly/patternfly-react#2179]
I would like to prerelease my changes in #2011 and begin testing downstream. There should be no breaking changes to `@patternfly/react-table`, however consumers should see the removal of Reactabular (`reactabular-table`) from their dependency tree.

Remaining things to circle back on later:

- [ ] tests
- [x] convert flow js to typescript where necessary
- [ ] a11y implementation for virtualized (t.b.d.)
- [x] ~fix Safari scrollbar in WindowScroller, this currently does not occur downstream (only in docs here)~ fixed in #2011, padding should be applied for WindowScroller scrollbar

**Comment by rachael-phillips** [comment]
@priley86 can we create two new issues. 1 for tests and 1 for the a11y work? The reason being we're hoping to assign these out to team members.

**Comment by priley86** [comment]
done. Opened #2328 and #2329. Feel free to adjust accordingly @rachael-phillips :)
